diff --git a/libvpx.gyp b/libvpx.gyp
index 4f8cb2b..1068de0 100644
--- a/libvpx.gyp
+++ b/libvpx.gyp
@@ -111,7 +111,7 @@
             ],
           },
           'dependencies': [
-            'gen_asm_offsets_vp8',
+            #'gen_asm_offsets_vp8',
           ],
           'includes': [
             '../yasm/yasm_compile.gypi'
@@ -225,8 +225,8 @@
           'target_name': 'libvpx',
           'type': 'static_library',
           'dependencies': [
-            'gen_asm_offsets_vp8',
-            'gen_asm_offsets_vpx_scale',
+            #'gen_asm_offsets_vp8',
+            #'gen_asm_offsets_vpx_scale',
           ],
 
           # Copy the script to the output folder so that we can use it with
